src/app/shared/utils/phone-helper.ts

```diff
@@ -0,0 +1,23 @@
+import { AccommodationOffer } from '@app/core/api';
+import { PREFIXES } from '../consts';
+import { PhoneNumber } from '../models/phone.model';
+
+export const PHONE_HELPER = {
+  initPhoneOnEdit: (context: any) => {
+    context.accommodationsResourceService.getAccommodations(context.offerId).subscribe((resp: AccommodationOffer) => {
```

Comment:
Tu coś się pomieszało ;), strzały do backendu powinny zostać w komponetach tak jak były wcześniej, tu można dorzucić metodę `findPrefix`

Reply:
Zrobiłem trochę inaczej, przekazuje parametr z typem do helpera i tam ogarniam całość zaciągania danych, myślę że wpłynie to pozytywnie na czytelność kodu.

Reply:
Ok, ale `PHONE_HELPER` -> `OFFER_HELPER` i `initPhoneOnEdit` -> `initOfferEdit`
I zastanawiam się czy nie lepiej zrezygnować z const i zrobić klasę OfferHelper

Reply:
Wstawiłem serwis: OfferDataInitService i faktycznie zmieniłem nazwę na initOfferDataForEdit